x/genutil/client/cli/init.go

```diff
@@ -81,14 +80,6 @@ func InitCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
 
 			serverCtx := server.GetServerContextFromCmd(cmd)
 			config := serverCtx.Config
-
-			// default mamaki configs TODO: move to the app to be more explicit
-			// in what we are setting as defaults
-			config.Consensus.TimeoutCommit = time.Second * 25
-			config.Consensus.SkipTimeoutCommit = false
-			config.Mempool.KeepInvalidTxsInCache = true
-			config.Mempool.TTLNumBlocks = 15
```

Comment:
We already have https://github.com/celestiaorg/celestia-app/blob/58352d17a16910ca7de94279b7a25c1ac5b0d791/cmd/celestia-appd/cmd/root.go#L81 so this doesn't need to be copied